vita3k/ngs/src/scheduler.cpp

```diff
@@ -87,10 +83,6 @@ bool VoiceScheduler::pause(Voice *voice) {
 }
 
 bool VoiceScheduler::resume(const MemState &mem, Voice *voice) {
-    if (voice->state != ngs::VOICE_STATE_PAUSED) {
-        return false;
```

Comment:
Im still not sure about this, the whole thing smell sus, you should add a note here about the game behaviour